lib/common/actions/index.js

```diff
@@ -183,7 +183,7 @@ const PatternLinkErrorMessage = (props) => {
         return (
           <li style={{listStyle: 'none', margin: '3px 0px'}}>
             <a href={patternPath}>
-              {`Route ${pattern.routeId}, Pattern ${pattern.patternId}`}
+              {`Route ${pattern.routeShortName}, Pattern ${pattern.patternId}`}
```

Comment:
This is not working with a number of feeds I previously imported. For these feeds, the route short name is undefined. You should fall back on route ID IMO.

Reply:
I agree, good call. I have added route ID as an fall back in https://github.com/ibi-group/datatools-ui/pull/985/commits/bfcde9e384936bb2ae673b4abe2ec6b2cfe82eeb